shared/graphql/schema.graphql

```diff
@@ -215,13 +215,24 @@ type User {
 type AccessCountForDate {
   date: Datetime!
   value: Int!
+  sum: Int!  # running sum
```

Comment:
not sure this is required, since it can be easily calculated on the consuming side, but it feels quite convenient to have this